src/gui/editors/PianoRoll.cpp

```diff
@@ -2867,6 +2867,8 @@ void PianoRoll::paintEvent(QPaintEvent * pe )
 
 	// set font-size to 8
 	p.setFont( pointSize<8>( p.font() ) );
+	QFontMetrics fontMetrics(p.font());
+	int const fontHeight = fontMetrics.boundingRect(QChar::fromLatin1('H')).height();
```

Comment:
The reason I used `- boundingRect.top() - boundingRect.bottom()` is that a font can span both the area above and the area below the baseline. It means using `height()` may give a different value for some fonts while the desired value is the center of the area.